pkg/util/defined_entity_util.go

```diff
@@ -61,18 +94,37 @@ func GetPVsFromRDE(rde *swaggerClient.DefinedEntity) ([]string, error) {
 	if !ok {
 		return nil, fmt.Errorf("unable to convert [%T] to slice of interface", pvInterfaces)
 	}
-	pvIdStrs := make([]string, len(pvInterfacesSlice))
+	//change list of PV-ID to list of PV-NAME
```

Comment:
sorry if i confused you before but I think we should not change the behavior of CSI to store PV names for native cluster. Since we used to store IDs in `persistentVolumes`, the RDE for clusters using CSI 1.1.2, for example, will have list of IDs. Unless we handle the existing values for `persistentVolumes` by changing IDs to names, we should not be  directly changing the PV IDs to names.